data/json/recipes/food/veggy.json

```diff
@@ -0,0 +1,219 @@
+[
+  {
+    "result": "mushroom_cooked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 0,
+    "time": 15000,
+    "batch_time_factors": [ 67, 5 ],
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 7 ], [ "char_smoker", 7 ], [ "toolset", 7 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "mushroom", 1 ] ]
+    ]
+  },
+  {
+    "result": "dahlia_baked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": 12000,
+    "reversible": false,
+    "autolearn": false,
+    "book_learn": [ [ "pocket_survival", 1 ], [ "survival_book", 1 ], [ "textbook_survival", 1 ], [ "manual_survival", 1 ], [ "survival_book", 1 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "dahlia_root", 1 ] ]
+    ]
+  },
+  {
+    "result": "potato_baked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": 15000,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 3 ], [ "char_smoker", 3 ], [ "toolset", 3 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "potato_raw", 1 ], [ "potato_irradiated", 1 ] ]
+    ]
+  },
+  {
+    "result": "veggy_cooked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 0,
+    "time": 4000,
+    "reversible": false,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "veggy", 1 ] ]
+    ]
+  },
+  {
+    "result": "veggy_wild_cooked",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 0,
+    "time": 4000,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "hotplate", 5 ], [ "char_smoker", 5 ], [ "toolset", 5 ], [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ], [ "aluminum_foil", 5 ] ],
+      [ [ "veggy_wild", 1 ] ]
+    ]
+  },
+  {
+    "result": "cooked_cattail_stalk",
```

Comment:
This was ported over from the recipe_food file, and originally changed to allow foil before Coolthulho advised against it.